index.html

```diff
@@ -4,285 +4,146 @@
-                    <li><img src="./image/Vector5.png" alt=""></li>
-                    <li><img src="./image/VectorGit.png" alt=""></li>
-                    <li><img src="./image/Linkedin.svg" alt=""></li>
-                </ul>
-            </div>
-        </div>
-        <div class="getme">
-            <button class="button1"><a>Get my resume </a>  </button>
-        </div>
-</div>
-<div> 
-    <div class="aboutprogram">
-        <ul class="Program-skil">
-            <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
-            <ul class="Program-skil">
-                <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
-                <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
-                <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
-            </ul>
-            <ul class="Programs-desclist">
-                <li class="flex">
-                    <img class="flex-img" src="image/descktop/JS.svg" alt="">
-                    <p>JavaSript</p>
-                  </li>
-                  <li class="flex">
-                    <img class="flex-img" src="image/descktop/Html.svg" alt="">
-                    <p>HTML</p>
-                  </li>
-                  <li class="flex">
-                    <img class="flex-img" src="image/descktop/Css.svg" alt="">
-                    <p>CSS</p>
-                  </li>
-            </ul>
-            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
-            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
-        </ul>
-    </div>
-</div>
-</section>
-<section class="cardcontact">
-    <form class="cardcontact1" action="https://formspree.io/f/mwkzayzq" method="post" >
-        
-        <div class="sectioncontact">
-            <h2  class="contactme" >Contact me </h2>
-            <p class="contact"> If you have an application you are interested 
-            in developing, a feature that you need built or a project that 
-            needs coding. I’d love to help with it  </p>
-        </div>
-        <fieldset class="filedset">
-            <input class="texform" type="text"  id="Name"   classe="form-control" placeholder="Alexandra Loremipsum" minlength="4" maxlength="30" required />
-            <input class="texform"  type="email"  id="email"  name="email"   placeholder="alexandra@gmail.com"  required  />
-           <input class="texform2" name="comment" id="comment" cols="50" rows="6" placeholder="Write your message here..." required >
-        </fieldset>
-            
-         <div class="buttonsee2" id="contactbuttonid">
-            <button class="contactbutton"> Get It Touch </button>
-        </div>
+    <section>
+        <h2>Featured Speakers</h2>
+        <div id="lineid" class="line"></div>
+        <section class="gridcontainer" id="Portfolio">
            
```

Comment:
- [ ] [OPTIONAL] Good job implementing the changes requested by the previous reviewer however your page should have three speakers in each column, kindly make sure to implement this